nautobot/extras/tests/test_models.py

```diff
@@ -1072,6 +1074,58 @@ def test_remote_url_hostname(self):
         self.repo.remote_url = "http://some-private-host/example.git"
         self.repo.validated_save()
 
+    def test_clone_context_manager(self):
+        """Confirm that the clone() context manager methods work as expected."""
+        self.tempdir = tempfile.TemporaryDirectory()  # pylint: disable=consider-using-with
```

Comment:
I don't see where you're cleaning up the tempdir when done - might be better to change it to context-manager style for the whole test case so that it cleans up automatically even if the test fails?